R/applyChangeMeta.R

```diff
@@ -194,27 +194,36 @@ recode_labels <- function(labels, changeTable, existingMeta) {
     values_being_recoded <- labels[single_simpleChanges[, "row_num_in_labels"], "value"]
 
     # relevant conflicts: existing values that are not themselves being recoded
+    ## tbd: create vector with ALL existing and being recoded values, even if they are being recoded into
+    ## then use this for ignore; figure out what else needs to be done in "value" and "value_new"
+
     existing_value_logical <- !is.na(single_simpleChanges$value_new) &
-      single_simpleChanges$value_new %in% single_labels$value &
-      !single_simpleChanges$value_new %in% values_being_recoded
+      single_simpleChanges$value_new %in% single_labels$value
     existing_value_vec <- single_simpleChanges$value_new[existing_value_logical]
+
+    existing_value_not_recoded_into_logical <- existing_value_logical &
+      !single_simpleChanges$value_new %in% values_being_recoded
+    existing_value_not_recoded_into_vec <- single_simpleChanges$value_new[existing_value_not_recoded_into_logical]

```

Comment:
Should these two objects be rather named something like `existing_value_not_being_recoded...`? I kept tripping over this while trying to understand the code - not least because `existing_value` already entails an implicit "that is in `value_new` (which I would interpret as being `recoded_into`)" if I am not mistaken. This could just be me, but if the name suggests something (slightly) different than is assigned to it, I would suggest changing it.
(I'm aware of #59 but this may save some time later)

Reply:
Yes, I think this makes the function easier to read. Thanks!